tpgtools/overrides/networkconnectivity/beta/spoke.yaml

```diff
@@ -0,0 +1,4 @@
+- type: CUSTOM_DESCRIPTION
```

Comment:
These should be upstreamed to the DCL rather than overrides specifically in Terraform I believe